ruby3.2-jrjackson.yaml

```diff
@@ -33,32 +23,14 @@ pipeline:
       repository: https://github.com/guyboertje/jrjackson
       tag: v${{package.version}}
 
-  - uses: maven/configure-mirror
-
-  - runs: |
-      gem install bundler
-      gem install ruby-maven
-      gem install maven-tools
-      gem install jar-dependencies
-
-  - runs: sed -i "s/3\.1/3.12.1/g; s/1\.8/11/g" Mavenfile
-
-  - runs: mvn clean install -Dmaven.test.skip
+  - uses: patch
+    with:
+      patches: mavenfile.patch
 
   - runs: |
-      # Extract the jackson_version to use in the maven dependency:copy commands
-      export JACKSON_VERSION=$(grep -A 1 'def self.jackson_version' ./lib/jrjackson/build_info.rb | tail -n 1 | awk -F "'" '{print $2}')
-
-      mkdir -p ./lib/com/fasterxml/jackson/core/jackson-annotations/${JACKSON_VERSION}
-      mkdir -p ./lib/com/fasterxml/jackson/core/jackson-core/${JACKSON_VERSION}
-      mkdir -p ./lib/com/fasterxml/jackson/core/jackson-databind/${JACKSON_VERSION}
-      mkdir -p ./lib/com/fasterxml/jackson/module/jackson-module-afterburner/${JACKSON_VERSION}
-      mkdir -p ./lib/jrjackson/jars
-
-      mvn dependency:copy -Dartifact=com.fasterxml.jackson.core:jackson-annotations:${JACKSON_VERSION} -DoutputDirectory=lib/com/fasterxml/jackson/core/jackson-annotations/${JACKSON_VERSION}
-      mvn dependency:copy -Dartifact=com.fasterxml.jackson.core:jackson-core:${JACKSON_VERSION} -DoutputDirectory=lib/com/fasterxml/jackson/core/jackson-core/${JACKSON_VERSION}
-      mvn dependency:copy -Dartifact=com.fasterxml.jackson.core:jackson-databind:${JACKSON_VERSION} -DoutputDirectory=lib/com/fasterxml/jackson/core/jackson-databind/${JACKSON_VERSION}
-      mvn dependency:copy -Dartifact=com.fasterxml.jackson.module:jackson-module-afterburner:${JACKSON_VERSION} -DoutputDirectory=lib/com/fasterxml/jackson/module/jackson-module-afterburner/${JACKSON_VERSION}
+      bundle install
+      rake
```

Comment:
```suggestion
      bundle exec rake
```

Reply:
Technically this should be executed in the context of the bundle. Practically in this case it won't matter, but it's a case of good practice